int_experian/int_experian_sfra/cartridge/static/default/js/edqUtils.js

```diff
@@ -333,6 +333,18 @@ for (var i = 0; i < buttonSelector.length; i++) {
 		buttonSelector[i].addEventListener("mouseover", edqEmailPhoneValidationCallback);
 	}
 }
+/** Added this function to set the listener tothe button on the selected stage (billing/shipping)
+* For more information see Bug #147798 */
+function setEdqButtonSelector(stageContentLocation) {
+	stageContentLocation = stageContentLocation || "";
+	if (stageContentLocation === "shipping") {
+		edqCurrentSubmitButtonSelector = document.querySelector("[value=submit-shipping]");
+		edqCurrentSubmitButtonSelector.addEventListener("mouseover", edqEmailPhoneValidationCallback);
+	} else if (stageContentLocation === "billing") {
+		edqCurrentSubmitButtonSelector = document.querySelector("[value=submit-payment]");
+		edqCurrentSubmitButtonSelector.addEventListener("mouseover", edqEmailPhoneValidationCallback);
```

Comment:
Was the intention to do the callback each time there is a mouseover?

Reply:
Yes, the idea is to create the callback when the stage is changed between Billing and Shipping.

Reply:
I see - however in this case the callback isn't run when the stage is changed, but rather when there is a mouseover. So for example if a user were to change pages with their keyboard would this callback still run?

Reply:
Changed the "mouseover" to "focus" event.

Reply:
Looks good. When we do the great refactor, one thing we should consider is using a MutationObserver instead of detecting button presses and what not. I'll explain more what this would look like when we're actually going to do it.